### PR TITLE
Update IMUs

### PR DESCRIPTION
**Pull request by kounocom**
This PR:

- Marks the BNO055 and all MPUs as not recommended for new designs.
- Adds BMI270 to components guide, schematics and IMU comparison
- Restructures `diy.js` IMU list from best to worst

I don't think marking BNO055/MPUs as not recommended is gonna be a controversial change, but I'm really unsure about the BMI270 part. More input is definitely needed, as there's massive conflict of interest for me to be documenting it while being the only person to sell these IMUs in module form.

### CODE REVIEW

**Comment by Hayash-i** [comment]
I personally 2nd the idea of stating that BNO055 / MPU should be listed as NRNDs since as we stand, MPU costs on par with BMI160/270 and is significantly worse in performance and with BNO, these cost significantly more then BMI and perform considerably worse, and solely in the interest of the end users' experience, BMI160 / 270 should be the go-to recommended IMU to provide the best price to performance and it has seemed to come upon the DIY channel to point people in the correct direction to use the better IMU, but it has been noted by my self and others that some users will not reference DIY first and buy MPU series IMU and then soon regret it providing a poor experience with SlimeVR.
In regards to Kounocom's schematic, I can confirm it is correct and is easily readable, Although I would like to suggest it mentioned that SDO must be soldered if using an extension as this is becoming an increasingly notable issue in the DIY channel, this is not BMI270 specific but all IMUs.
I would also suggest other methods of the BMI270 being acquired be mentioned in the docs, such as creating your own PCB design or commissioning others to design a PCB for them as I am aware due to some postage costs, ordering from Kounocom might not be feasible.
I look forward to the BMI270 being added to the main documents.

File: src/assets/js/diy.js [hunk 1]
```diff
@@ -26,53 +26,60 @@
...
                 },
                 {
-                    'name': 'MPU9250',
+                    'name': 'BNO055',
                     'amount': (set) => set,
-                    'cost': 4.75,
-                    'costAll': (set) => set * 4.75 + 2.73,
-                    'links': '<a href="https://www.aliexpress.com/wholesale?SearchText=MPU9250">AliExpress MPU9250</a>. Buyer beware: large amount of fakes.'
+                    'cost': 55.10,
+                    'costAll': (set) => set * 55.10 + 2.73,
```
Comment (ButterscotchV) [inline src/assets/js/diy.js:67]: This definitely isn't accurate anymore if you are looking to update more :P

File: src/diy/imu-comparison.md [hunk 2]
```diff
@@ -45,10 +46,31 @@ Score: <i class="fa fa-star"></i><i class="fa fa-star"></i><i class="fa fa-star"
...
+## BMI270
+The BMI270 is a relatively new and <b>experimental</b> IMU for DIY SlimeVR.
+It seems to perform significantly better than the BMI160 while still being affordable.
+
+|Reset time |Cost  |Availability|Build quality|
+|:---------:|:----:|:----------:|:-----------:|
+|20 - 30min |~$3.8 |Sufficient  |Great        |
+
+note: what do I even put for score?
+Score: ?????
```
Comment (ButterscotchV) [inline src/diy/imu-comparison.md:59]: I guess just BMI160 + 1, idk

**Review by ButterscotchV — COMMENTED** [review]
Looks alright, we could probably also add LSM as it seems to be coming up more, though I'm not sure about any of the information for it

File: src/diy/imu-comparison.md (hunk 2 above)
Comment (kounocom) [reply src/diy/imu-comparison.md]: I put it at 4 stars for now

File: src/assets/js/diy.js (hunk 1 above)
Comment (kounocom) [reply src/assets/js/diy.js]: I set it to $17, which is a good average of ali prices I think. Not that anyone should use these :P

**Comment by kounocom** [comment]
> Looks alright, we could probably also add LSM as it seems to be coming up more, though I'm not sure about any of the information for it

I'm hesitant to document the LSM6DSV just yet, the firmware for it needs some fixing, and it still needs a lot more testing. Keeping stock of it is also an issue for me.

**Review by ButterscotchV — APPROVED** [review]

**Pull request merged**